jest.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  collectCoverageFrom: ['src/components/**/*.js?(x)', '!src/**/*.story.js?(x)'],
+  collectCoverageFrom: [
+    'src/components/**/*.js?(x)',
+    '!src/**/*.story.js?(x)',
+    '!src/**/hooks/*.js',
```

Comment:
Note: The hook (merged in a previous PR) wasn't being included in the per-file coverage report, but was being included in the global count.  This hook is a reference implementation (closer to a story, or AsyncTable, than an actual component) so I recommend we defer coverage until we have a larger strategy for how the PAL will maintain and export data hooks.

Reply:
Yeah, fair. The related component is under experimental anyway. I've opened https://github.com/carbon-design-system/carbon-addons-iot-react/issues/1615 to track work towards guidance on unit testing hooks we export and reimplementing the testing thresholds.

Reply:
Actually just realized the related component isn't under experimental, but we should be fine as this is directly for your team to consume.